stdlib/public/core/StringIndex.swift

```diff
@@ -368,7 +368,8 @@ extension String.Index {
 extension String.Index {
   @_alwaysEmitIntoClient // Swift 5.7
   internal func _copyingAlignment(from index: Self) -> Self {
-    let mask = Self.__scalarAlignmentBit | Self.__characterAlignmentBit
+    let mask = Self.__scalarAlignmentBit
+      | Self.__characterAlignmentBit
```

Comment:
What's this change for?

Reply:
It was leftover when I had wordAlignmentBit